docs/modules/ROOT/pages/integrating-with-existing-deployments.adoc

```diff
@@ -0,0 +1,240 @@
+
+. At least one **Couchbase Server** node;
+. **Prometheus**, with **Alertmanager** installed;
+. **Grafana**, configured to use the Prometheus server as a Data Source;
+- **Docker**, to run the Cluster Monitor's Docker image
+
+== Installation
+
+=== Install Cluster Monitor
+
+The Cluster Monitor can be installed by downloading a Docker image.
+
+[NOTE]
+====
+We plan for the image to be distributed on Docker Hub, but currently this is not available. 
+====
+
+Run the Docker container, using the preview image given to you by Couchbase:
+[source, console]
+----
+docker run -d --rm \
+  -p 7196:7196 \ <.>
+  -e CB_MULTI_ALERTMANAGER_URLS=http://<Alertmanager IP>:9093 \ <.> 
+  -e CB_MULTI_ADMIN_USER=admin -e CB_MULTI_ADMIN_PASSWORD=password \
+  --name cluster_monitor \
+  <YOUR IMAGE HERE>
+----
+
+<.> If you are using TLS, you will need to add `-p 7197:7197` to also expose port `7197`.
+<.> This is not necessary if you are not using Alertmanager
+
+[NOTE]
+====
+Docker networking https://docs.docker.com/network/#network-drivers[defaults to bridge], meaning that this container will be able to communicate with any other running Docker containers.
+If your Grafana, Prometheus / Alertmanager instances are not containerized, ensure that the Cluster Monitor container can communicate with them.
+====
+=== Configure Cluster Monitor
+
+Navigate to the Web GUI, which listens by default on port `7196`.
+
+Then click "Add Cluster" and enter the IP address of a node in the cluster, along with the cluster's configured username and password.
+Repeat this for every cluster you wish to add to the Cluster Monitor.
+
+=== Install the Agent on your Couchbase Server nodes
+
+See https://issues.couchbase.com/browse/CMOS-210[CMOS-210], which tracks the development (including documentation) of the Agent.
+Until this is ready, follow the instructions given to you by your Couchbase representative.
+
+== Configuration
+
+=== Grafana: Plugins
+
+[NOTE]
+====
+The instructions for each plugin offer a Grafana Cloud one-click install, a command-line based install for a running instance, or a `.zip` file which can be unpacked manually into your Grafana plugins directory. 
+====
+
+**Currently, we require only https://grafana.com/grafana/plugins/marcusolsson-json-datasource/[JSON API], which has associated https://grafana.com/grafana/plugins/marcusolsson-json-datasource/?tab=installation[installation instructions].**
+
+However, if you:
+
+- Configure Grafana (v7.1+) dashboards through the use of provisioning files: https://grafana.com/docs/grafana/latest/administration/provisioning/#plugins[specify the plugins to install in your provisioning configuration file]. 
+- Are running Grafana in a Docker container: simply pass through an additional environment variable `GF_INSTALL_PLUGINS=$PLUGIN_NAME $VERSION` where the `$PLUGIN_NAME` and latest `$VERSION` can be found on the plugin's homepage (linked under "installation instructions" for each). 
```

Comment:
Should we have `grafana-cli plugin install` as a bullet-pointed step here? It's the easiest way in a standalone environment, and, even though we do link out to Grafana's docs, in my head it doesn't quite make sense to mention some of the possible ways but omit the easiest one.

Reply:
That's contained within the "main" installation instructions which are https://grafana.com/grafana/plugins/marcusolsson-json-datasource/?tab=installation (referenced in the intro as "command-line based install").

These are just alternatives.

Reply:
I saw that, but that link is rather small when there's two much bigger bullet points right below it giving you what looks like the two ways to install it - IMHO it should be either listing all of the common ones, or omitting these bullet points at all (since they'll be covered in grafana.com anyway)

Reply:
Sure, only makes sense to compact when there's multiple plugins but there's only one. Will change.

Reply:
(tbh I'm leaning towards "here's a list of plugins you need, here's the link to grafana.com with how to install them, go wild" - though if you think it's beneficial to have explicit steps I'll happily defer to you on that)

Reply:
The problem is the official documentation doesn't include the steps for Docker or its own provisioning files (at least, for the plugins' installation steps).